README.md

```diff
@@ -12,6 +12,8 @@ To learn more about why we created and maintain this boilerplate project, read o
 
 ## Usage
 
+Make sure that `yarn` is correctly configured before using the boilerplate. Otherwise, you might encounter some packaging errors when using other tools like `npm`.
```

Comment:
> when using other tools like `npm`.

Should we mention that `yarn` should always be used when executing commands instead of `npm `? (eg. `yarn foo bar` instead of `npm foo bar`)

Reply:
Done! Even extracted this in its own section of the doc

Reply:
Does `npx` use `npm` by default? Is there a `yarn` alternative?

Reply:
Mmmm good question, I never had issues running `npx` in the past, but I never ran it for stuff depending on yarn dependencies so I'm not sure